Dockerfile

```diff
@@ -63,15 +65,14 @@ RUN apt-get update -qq && apt-get install -y \
 
 WORKDIR /near/indexer-explorer
 
-COPY --from=build /usr/local/cargo/bin/diesel .
 COPY --from=build /tmp/target/release/indexer-explorer .
-# Diesel needs a migrations directory to run
-COPY --from=build /near/indexer-explorer/migrations ./migrations
- 
-# If the --store-genesis flag isn't set, the accounts in genesis won't get created in the DB which will lead to foreign key constraint violations
-# See https://github.com/near/near-indexer-for-explorer/issues/167
-CMD ./diesel migration run && \
-    ./indexer-explorer --home-dir /root/.near/localnet init ${BOOT_NODES:+--boot-nodes=${BOOT_NODES}} --chain-id localnet && \
-    sed -i 's/"tracked_shards": \[\]/"tracked_shards": \[0\]/' /root/.near/localnet/config.json && \
-    sed -i 's/"archive": false/"archive": true/' /root/.near/localnet/config.json && \
-    ./indexer-explorer --home-dir /root/.near/localnet run --store-genesis sync-from-latest
+
+COPY --from=build /usr/local/cargo/bin/diesel .
+# NOTE: We'd expect to need the diesel.toml here, but for some reason we don't
```

Comment:
`diesel.toml` contains settings that are used during the generation of `schema.rs` it's not involved in the migration process, your expectation here is incorrect :)

Reply:
Removed!